tests/test_csc.py

```diff
@@ -160,6 +160,107 @@ def test_standard_state_transitions(self):
+            self.assertEqual(state.summaryState, salobj.State.STANDBY)
+            # send start; new state is DISABLED 
+            cmd_attr = getattr(harness.remote, f"cmd_start")
+            state_coro = harness.remote.evt_summaryState.next(timeout=1.)
+            id_ack = await cmd_attr.start(cmd_attr.DataType(), timeout=1.)
+            state = await state_coro
+            self.assertEqual(id_ack.ack.ack, harness.remote.salinfo.lib.SAL__CMD_COMPLETE)
+            self.assertEqual(id_ack.ack.error, 0)
+            self.assertEqual(harness.csc.summary_state, salobj.State.DISABLED)
+            self.assertEqual(state.summaryState, salobj.State.DISABLED)
+            # set to FAULT; new state is FAULT
+            harness.csc._summary_state = salobj.State.FAULT
+            state_coro = harness.remote.evt_summaryState.next(timeout=1.)
+            harness.csc.report_summary_state()
+            state = await state_coro
+            self.assertEqual(harness.csc.summary_state, salobj.State.FAULT)
+            self.assertEqual(state.summaryState, salobj.State.FAULT)
+
+
+            # send standby; new state is STANDBY
+            cmd_attr = getattr(harness.remote, f"cmd_standby")
+            state_coro = harness.remote.evt_summaryState.next(timeout=1.)
+            id_ack = await cmd_attr.start(cmd_attr.DataType(), timeout=1.)
+            state = await state_coro
+            self.assertEqual(id_ack.ack.ack, harness.remote.salinfo.lib.SAL__CMD_COMPLETE)
+            self.assertEqual(id_ack.ack.error, 0)
+            self.assertEqual(harness.csc.summary_state, salobj.State.STANDBY)
+            self.assertEqual(state.summaryState, salobj.State.STANDBY)
+            # send start; new state is DISABLED 
+            cmd_attr = getattr(harness.remote, f"cmd_start")
+            state_coro = harness.remote.evt_summaryState.next(timeout=1.)
+            id_ack = await cmd_attr.start(cmd_attr.DataType(), timeout=1.)
+            state = await state_coro
+            self.assertEqual(id_ack.ack.ack, harness.remote.salinfo.lib.SAL__CMD_COMPLETE)
+            self.assertEqual(id_ack.ack.error, 0)
+            self.assertEqual(harness.csc.summary_state, salobj.State.DISABLED)
+            self.assertEqual(state.summaryState, salobj.State.DISABLED)
+            # send enable; new state is ENABLED 
+            cmd_attr = getattr(harness.remote, f"cmd_enable")
+            state_coro = harness.remote.evt_summaryState.next(timeout=1.)
+            id_ack = await cmd_attr.start(cmd_attr.DataType(), timeout=1.)
+            state = await state_coro
+            self.assertEqual(id_ack.ack.ack, harness.remote.salinfo.lib.SAL__CMD_COMPLETE)
+            self.assertEqual(id_ack.ack.error, 0)
+            self.assertEqual(harness.csc.summary_state, salobj.State.ENABLED)
+            self.assertEqual(state.summaryState, salobj.State.ENABLED)
+            # set to FAULT; new state is FAULT
+            harness.csc._summary_state = salobj.State.FAULT
```

Comment:
Same as in [here](https://github.com/lsst-ts/ts_scheduler/pull/19/files#diff-d9365181a6fb138131218b600a0dfccfR191)